assets/js/components/ExecutionResults/ExecutionResults.test.jsx

```diff
@@ -87,7 +108,8 @@ describe('ExecutionResults', () => {
     const {
       clusterID,
       hostnames,
-      checkID,
+      checkID1,
```

Comment:
Should we return maybe `checks`? As we do with the hostnames

Reply:
Fair point